(feat) Konnect in GCP Changelog (#6650)
Kong/docs.konghq.com: app/konnect/updates.md

services. [Try it today!](https://cloud.konghq.com/quick-start)

## December 2023
**Konnect in Google Cloud Marketplace**

> : {{site.konnect_saas}} is now available for purchase through the Google Cloud (GCP) Marketplace including public and private offers. This allows Kong's customers to retire their existing GCP Credits through their purchase of {{site.konnect_saas}} and associated product capabilities.

Reviewer: ```suggestion
: {{site.konnect_saas}} is now available for purchase through the Google Cloud (GCP) Marketplace including public and private offers. This allows you to retire your existing GCP Credits through your purchase of {{site.konnect_saas}} and associated product capabilities.
```
